README.md

```diff
@@ -2,22 +2,54 @@
 
 A web crawler to discover UK-based domestic violence support services.
 
+## Requirements
+
+ * Charity Register, stored in `resource` folder, with path updated in the `__SCOT_CHARITIES_PATH` variable in `heuristics/dvdvsc_scorers.py`. Retrieved from the [OSCR](https://www.oscr.org.uk/about-charities/search-the-register/download-the-scottish-charity-register/).  © Crown Copyright and database right [2025]. Contains information from the Scottish Charity Register supplied by the Office of the Scottish Charity Regulator and licensed under the Open Government Licence v.3.0. (http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/)
+
 ## Use with Docker
 
-After building the image with `docker compose build`, run `docker compose up` to start three containers:
+After building the image with `docker compose build`, run `docker compose up` to start four containers:
 
 * `app` (the crawler)
 * `db` (the PostgreSQL database server)
 * `pgadmin`
+* `ollama` (requires nvidia GPU)
 
 This automatically loads environment variables from `.env` in the project directory. Set one up as per `example.env`.
 
-`docker compose run db pgadmin` will only spin up the database and pgAdmin containers. Access pgAdmin from a browser at port 5051, as specified in `compose.yaml`.
+To ensure use of `schema_dump.sql` to initialise database, need to remove `db_data` volume using `docker volume rm st-andrews-dvsvc_db_data` before using `up` command.
+
+`docker compose run db pgadmin` will only spin up the database and pgAdmin containers. Access pgAdmin from a browser at port 5051, as specified in `compose.yaml`. The hostname of the database will be the container ID of db container.
 
 ## Run just the crawler (without Docker)
+First, set up an virtual environment, with requirements.txt installed (on Lab PCs use psycopg2-binary instead of psycopg2).
```

Comment:
Would be as explicit as possible with these steps. Imagine me trying to follow these steps. (I've never setup a virtual environment). 

Or imagine Annah/Bobby/David trying to do this. Give all code/commands that isn't there explicitly please! 

Or if possible wrap it all into one script and get us to run that. :)